converter/include/k4SimDelphes/DelphesEDM4HepConverter.h

```diff
@@ -51,6 +52,13 @@ namespace k4SimDelphes {
    */
   constexpr auto CALORIMETERHIT_OUTPUT_NAME = "CalorimeterHits";
 
+  /**
+   * * Eventheader class will be stored only
+   */
+
+  constexpr auto EVENTHEADER_NAME = "EventHeader";
+
```

Comment:
```suggestion
  /**
   * * Eventheader class will be stored only
   */
  constexpr auto EVENTHEADER_NAME = "EventHeader";
```
To make formatting happy